Games/core/Game.js

```diff
@@ -791,12 +794,48 @@ module.exports = class Game {
         this.processAlertQueue();
         this.events.emit("meetingsMade");
 
+        this.vegMeeting = undefined;
+
         // Create next state timer
         this.createNextStateTimer(stateInfo);
     }
 
     createNextStateTimer(stateInfo) {
-        this.createTimer("main", stateInfo.length, () => this.gotoNextState());
+        this.createTimer("main", stateInfo.length, () => this.checkVeg());
+        this.checkAllMeetingsReady();
+    }
+
+    checkVeg() {
+        this.clearTimer("main");
+        this.clearTimer("secondary");
+        this.vegMeeting = this.createMeeting(VegReadyMeeting, "vegKickMeeting");
+        var vegMeetings = Object.values(this.history.states[this.currentState].meetings).filter(x => x.id !== this.vegMeeting.id);
+        for (let player of this.players) {
+            var addToKick = true;
+            if (player.alive) {

```

Comment:
a guardclause seems better here?

if (!player.alive) 
  continue

addToKick = true;
// iterate meetings, check if player is eligible to add to kick
xxx